mailscanner/auto-release.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * Mailwatch for Mailscanner Modification
+ * Author: Alan Urquhart - ASU Web Services Ltd
+ * Version: 1.0
+ * Date: 12-04-2016
+ *
+ * Requires: Mailwatch 1.2.0
+ *
+ * Provides the mechanism for one click release of quarantined emails as reported by the quarantine_report.php cron
+ *
+ * SETUP:
+ *
+ * Create the following table in the mailscanner database:
+ * CREATE TABLE `mod_release` (
+ *  `id` int(11) NOT NULL AUTO_INCREMENT,
+ *  `msg_id` varchar(255) NOT_NULL,
+ *  `uid` varchar(255) NOT_NULL,
+ *  PRIMARY_KEY (`id`)
+ * );
+ *
+ * Update cron.daily/quarantine_report.php with the modified file
+ * Update cron.daily/quarantine_maint.php with the modified file
+ *
+ * Create a new database user with limited privileges on the mod_release table - minimum is SELECT,DELETE
+ * Enter database credentials below.
+ */
+if (isset($_GET['mid']) && isset($_GET['r'])) {
+    // Change the following to reflect the location of functions.php
+    require_once('/var/www/html/mailscanner/functions.php');
+    //Database Credentials
+    $host = 'localhost';//change if using a remote db
+    $user = '';
+    $pass = '';
+    $database = 'mailscanner'; //change is your database is called something else
+    $db = mysqli_connect($host, $user, $pass, $database) or die("CONNECT ERROR" . mysqli_connect_error());
+    $mid = mysqli_real_escape_string($db, $_GET['mid']);
+    $token = mysqli_real_escape_string($db, $_GET['r']);
+    $query = "SELECT * FROM mod_release WHERE msg_id = '$mid'";
+    $result = mysqli_query($db, $query);
```

Comment:
I know that it sounds anacronistic in 2016, bit alla mailwatch 1.2 code uses MySQL estention...